source/Coop/Mod/Persistence/MobilePartyEntityManager.cs

```diff
@@ -116,7 +116,7 @@ private void InitRoom(RailServerRoom room)
             CampaignEvents.OnPartyRemovedEvent.AddNonSerializedListener(this, OnPartyRemoved);
             
             CampaignEvents.MobilePartyCreated.AddNonSerializedListener(this, OnPartyAdded);
-            CampaignEvents.OnLordPartySpawnedEvent.AddNonSerializedListener(this, OnPartyAdded);
+            LordPartyComponentPatch.OnLordPartySpawned += (sender, e) => OnPartyAdded(e);
```

Comment:
Might be able to just get away with removing this line wholly rather than adding `LordPartyComponentPatch`. Since 1.5.9 changed it so that lord parties are just mobile parties, we would probably be handling the party being added twice for players. (Might be a reason why the main party spawned and AI took control of it?)

Reply:
Removing the patch indeed fixed the duplicate party. Thanks!